src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java

```diff
@@ -598,23 +595,4 @@ public void testAllAttributesAddedToSuperUser() {
                 times(1)).saveAll(any());
     }
 
-    @Test
-    public void testFakeAttributesNotAdded() {
-        prdEnums.add(new PrdEnum(new PrdEnumId(0, "FAKE"), "pui-fake-manager", "FAKE_ROLE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(1, "FAKE_ROLE"), "pui-fake-manager", "FAKE_ROLE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(2, "FAKE_FAKE"), "pui-fake-manager", "FAKE_ROLE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(10, "JURISD_ID"), "pui-fake-manager", "FAKE_ROLE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(10, "FAKE_JURISD"), "PROBATE", "FAKE_ROLE"));
-
-
-        List<UserAttribute> attributes = new ArrayList<>();
-        attributes.add(userAttributeMock);
-        when(prdEnumRepositoryMock.findAll()).thenReturn(prdEnums);
-        when(prdEnumServiceMock.findAllPrdEnums()).thenReturn(prdEnums);
-        when(userAttributeRepositoryMock.saveAll(any())).thenReturn(attributes);
-
-        organisationServiceImplMock.createOrganisationFrom(organisationCreationRequest);
-
-        verify(userAttributeRepositoryMock, times(0)).saveAll(any());
-    }

```

Comment:
Again, I believe this was test was created to fulfil a mutation scenario

Reply:
Hi this was an error and will be added back

Reply:
nice 😄